theseus_gui/src/components/SearchPanel.vue

```diff
@@ -0,0 +1,81 @@
+<script setup>
+import { ref } from 'vue'
+import {
+  SearchIcon,
+  Button,
+  BoxIcon,
+  BookIcon,
+  ContractIcon,
+  TransferIcon,
+  ServerIcon,
+} from 'omorphia'
+import { useInstances } from '@/store/instancesStore'
+
+const instanceStore = useInstances()
+const searchText = ref('')
+
+const searchHandler = async () => {
+  instanceStore.setSearchInput(searchText.value)
+  await instanceStore.searchInstances()
+}
+</script>
+
+<template>
+  <div class="search-panel-container">
+    <div class="search-panel">
+      <div class="iconified-input">
+        <SearchIcon />
+        <input type="text" placeholder="Search.." v-model="searchText" @input="searchHandler" />
+      </div>
+      <Button>Search by relevance</Button>
+    </div>
+    <!-- Open a modal onClick to filter these things? -->
+    <div class="filter-panel">
+      <Button><BoxIcon />Minecraft version</Button>
+      <Button><ContractIcon />Categories</Button>
+      <Button><ServerIcon />Environment</Button>
+      <Button><TransferIcon />Mod loaders</Button>
+      <Button><BookIcon />License</Button>
+    </div>
```

Comment:
These filters should be moved over to a right or left sidebar similar to what we have on the main site